dapr-spring/dapr-spring-boot-tests/src/main/java/io/dapr/spring/boot/testcontainers/service/connection/DaprContainerConnectionDetailsFactory.java

```diff
@@ -0,0 +1,45 @@
+package io.dapr.spring.boot.testcontainers.service.connection;
+
+import io.dapr.spring.boot.autoconfigure.client.DaprConnectionDetails;
+import io.dapr.testcontainers.DaprContainer;
+import org.springframework.boot.testcontainers.service.connection.ContainerConnectionDetailsFactory;
+import org.springframework.boot.testcontainers.service.connection.ContainerConnectionSource;
+
+public class DaprContainerConnectionDetailsFactory
+        extends ContainerConnectionDetailsFactory<DaprContainer, DaprConnectionDetails> {
+
+  DaprContainerConnectionDetailsFactory() {
+  }
+
+  protected DaprConnectionDetails getContainerConnectionDetails(ContainerConnectionSource<DaprContainer> source) {
+    return new DaprContainerConnectionDetails(source);
+  }
+
```

Comment:
Could we rename this to `DefaultDaprConnectionDetails`, `DaprContainer` seems redundant. We want to connect to Dapr, irrespective where it is running, container or not.

Reply:
The naming convention comes from this: DaprContainer paramter